pyfr/plugins/fluidforce.py

```diff
@@ -172,6 +201,15 @@ def __call__(self, intg):
 
                 # Do the quadrature
                 f[ndims:] += np.einsum('i...,klij,jil', qwts, vis, norms)
+                if self._mcomp:
+                    # Viscous force at each flux point
+                    viscf = np.einsum('ijkl,lkj->lki', vis, norms)
+
+                    # Viscous force moments
+                    mop = 'i,ji' if self.ndims == 2 else 'i,jik'
+                    m[self._mcomp:] += np.einsum(mop, qwts,
+                                                 np.cross(rfpts, viscf)
+                                                 ).sum(axis=0)
```

Comment:
Can einsum not handle the .sum(axis=0) part?

Reply:
Fixed it.